exampleCourse/questions/elementDropdown/server.py

```diff
@@ -0,0 +1,28 @@
+import random, copy
+
+def generate(data):
+
+	QUESTION1 = 'aristotle'
+	QUESTION2 = 'socrates'
+	QUESTION3 = 'hume'
+
+	# Can override pl-answer options as parameters
+	data['params'][QUESTION1] = { 
+		'tag1': 'true', 'ans1': 'whole', 
+		'tag2': 'false', 'ans2': 'part',
+		'tag3': 'false', 'ans3': 'inverse' 
+	}
+
+	data['params'][QUESTION2] = {
+		'tag1': 'false', 'ans1': 'insatiable',
+		'tag2': 'true', 'ans2': 'unexamined',
+		'tag3': 'false', 'ans3': 'examined' 
+	}
+
+	data['params'][QUESTION3] = {
+		'tag1': 'true', 'ans1': 'wise',
+		'tag2': 'false', 'ans2': 'clumsy',
+		'tag3': 'false', 'ans3': 'reckless'
+	}
+
+	return data
```

Comment:
Missed a newline here:

```suggestion
	return data
	
```

Reply:
Just curious if we can add this to the linting rules. It has proven quite effective so far.

Reply:
Great point, let me look into that :)